bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -1,40 +1,13 @@
 package social.bigbone
 
-import com.google.gson.Gson
-import okhttp3.HttpUrl
-import okhttp3.Interceptor
+import kotlinx.serialization.json.Json
+import okhttp3.*
```

Comment:
We discourage star imports in this project, so this should be changed. Actually, this should popup as a warning as well, if you run `gradle check`. Let me know if this is not the case.

Reply:
Ah, then that’s the first very obvious clash of the default style applied in IntelliJ. 😊 I’ll check.

Reply:
Eventually, I will try to create a config file for IntelliJ then. Thanks for pointing this out.

Reply:
I have created an issue for this. Let's collect all the deviations there: https://github.com/andregasser/bigbone/issues/279